File: DMOJ/DMOJ/16bit/16bit.cpp
#include <stdio.h>

int main() {
	int n, a, b, c;
	scanf("%d", &n);
	while (n-- > 0) {
		scanf("%d %d %d", &a, &b, &c);
		printf(a * b == c ? "POSSIBLE DOUBLE SIGMA\n" : "16 BIT S/W ONLY\n");
	}
}

File: DMOJ/DMOJ/16bit/16bit.py
numLines = int(input())
arr = []

for i in range(numLines):
    temp = input().split()
    if int(temp[1])*int(temp[0]) != int(temp[2]):
        arr.append("16 BIT S/W ONLY")
    else:
        arr.append("POSSIBLE DOUBLE SIGMA")

for i in arr:
    print(i)
